package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "redux-form-inspector",
+  "description": "React HOC which add inspection capabilities to redux-form instances",
+  "version": "0.0.1",
+  "license": "MIT",
+  "authors": [
+    "Demangeon Julien <julien@marmelab.com>"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/marmelab/redux-form-inspector"
+  },
+  "bugs": {
+    "url": "https://github.com/marmelab/redux-form-inspector/issues"
+  },
+  "main": "lib/index.js",
+  "scripts": {
+    "prepublish": "make build"
+  },
+  "devDependencies": {
+    "babel-cli": "6.24.1",
+    "babel-core": "6.24.1",
+    "babel-eslint": "7.2.3",
+    "babel-plugin-transform-object-rest-spread": "6.23.0",
+    "babel-preset-es2015": "6.24.1",
+    "babel-preset-react": "6.24.1",
+    "enzyme": "2.8.2",
+    "eslint": "3.19.0",
+    "eslint-config-airbnb": "15.0.0",
+    "eslint-plugin-import": "2.2.0",
+    "eslint-plugin-jsx-a11y": "5.0.1",
+    "eslint-plugin-mocha": "4.9.0",
+    "expect": "1.20.2",
+    "mocha": "2.5.3",
+    "react-addons-test-utils": "15.5.1"
+  },
+  "dependencies": {
+    "redux": "3.6.0",
+    "react": "15.5.4",
+    "react-dom": "15.5.4",
+    "react-redux": "^5.0.1",
```

Comment:
:+1:
But the maintainer should keep his library up-to-date following the update of the "big" libraries.